atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/OracleTableNameUnmapper.java

```diff
@@ -51,7 +51,7 @@ public String getShortTableNameFromMappingTable(
             return unmappingCache.get(fullTableName, () -> {
                 SqlConnection conn = null;
                 try {
-                    conn = connectionSupplier.getNewUnsharedConnection();
+                    conn = connectionSupplier.get();
```

Comment:
Oracle tests live internally in `dbkvs-oracle-driver`, hence the tests passed here. I will pick reviewing this first thing tomorrow morning.